.github/workflows/nightly-normal-session-e2e-tests.yml

```diff
@@ -371,6 +371,31 @@ jobs:
           artifact-aleph-e2e-client-image: ${{ needs.build-aleph-e2e-client-image.outputs.artifact-name-image }}
           timeout-minutes: 35
 
+  run-e2e-all-validators-have-ideal-performance:
+    needs:
+      - build-synthetic-node
+      - build-aleph-e2e-client-image
+      - build-chain-bootstrapper-production
+    name: All validators have ideal abft performance
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v4
+
+      - name: Run e2e test
+        uses: ./.github/actions/run-e2e-test
+        with:
+          test-case: all_validators_have_ideal_performance
+          # yamllint disable-line rule:line-length
+          artifact-aleph-node-image: ${{ needs.build-synthetic-node.outputs.artifact-name-image }}
```

Comment:
We should have here normal binary, ie `${{ needs.build-production-aleph-node.outputs.artifact-name-image }}`

Reply:
done